### ISSUE TITLE
clean macros setting doesn't affect resharper intellisense

### ISSUE THREAD

**Issue by bradphelan** [jaredpar/VsVim#1886]
The clean macro feature is great. However it fails to turn off autocomplete/intellisense when resharper is driving

**Comment by jaredpar** [comment]
Yeah unfortunately R# intellisense operates through a very different API / command chain.  It's much harder to disable from a Visual Studio extension.  Will see if I can work out a way to make this happen.

**Comment by christianrondeau** [comment]
Oops, I commented on #1629 - I see there was a discussion with JetBrains a few years back: https://resharper-support.jetbrains.com/hc/en-us/community/posts/206657885-Detecting-and-dismissing-Intellisense-from-an-extension maybe they could be of help?
